vuestorefront/schemas/website_page.py

```diff
@@ -40,49 +28,55 @@ class WebsitePageFilterInput(graphene.InputObjectType):
 
 class WebsitePageQuery(graphene.ObjectType):
     website_page = graphene.Field(
-        WebsitePageI,
+        WebsitePage,
         required=True,
-        id=graphene.Int(),
-        content_rendering=graphene.Argument(WebsitePageContentInput, default_value={})
+        id=graphene.Int(default_value=None),
+        name=graphene.Int(default_value=None),
+        url=graphene.Int(default_value=None),
```

Comment:
@oerp-odoo I see that we missed all these during the PR but I guess for `name` and `url` shouldn't be used `Int`, right?

Reply:
Yes, thats a typo.. Will fix it.